server/src/metadata/tenant-migration/tenant-migration.service.ts

```diff
@@ -62,7 +62,7 @@ export class TenantMigrationService {
     workspaceId: string,
   ): Promise<TenantMigrationEntity[]> {
     return await this.tenantMigrationRepository.find({
-      order: { createdAt: 'ASC' },
+      order: { name: 'ASC', createdAt: 'ASC' },
```

Comment:
@martmull createdAt should be first here and should be the source of truth and name should came later to help with the order. To add to that, we have name that can be empty, for customMigrations (they don't rely on a file so there is no name for them) which means it will first oder by name and put all custom migrations at the top THEN order by createdAt. The opposite should be safer
```suggestion
      order: { createdAt: 'ASC', name: 'ASC' },
```

Reply:
cc @charlesBochet

Reply:
Fine